extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js

```diff
@@ -4192,6 +4210,9 @@ var globalRequire = require;
 					var adErrorEvent = google.ima.AdErrorEvent.Type;
 					_IMA_SDK_destroyAdsLoader();
 					_IMAAdsLoader = new google.ima.AdsLoader(_IMAAdDisplayContainer);
+					// This will enable notifications whenever ad rules or VMAP ads are scheduled
+					// for playback, it has no effect on regular ads
+					_IMAAdsLoader.getSettings().setAutoPlayAdBreaks(false);
 					_IMAAdsLoader.addEventListener(adsManagerEvents.ADS_MANAGER_LOADED, _onAdRequestSuccess, false);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the declaration of "_onAdRequestSuccess" before this usage. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)